source/codegen/metadata/nidmm/attributes.py

```diff
@@ -308,7 +308,7 @@
     1150044: {
         'access': 'read only',
         'channel_based': False,
-        'name': 'FREQ_VOLTAGE_AUTO_RANGE',
+        'name': 'FREQ_VOLTAGE_AUTO_RANGE_VALUE',
```

Comment:
What is the attribute name is attributes.hapigen file in the base metadata folder?

Reply:
the attribute name in attributes.hapigen is `freq voltage auto range`